oshi-core/src/main/java/oshi/util/ParseUtil.java

```diff
@@ -38,6 +39,7 @@
  * String parsing utility.
  */
 @ThreadSafe
+@SuppressForbidden(reason = "Require parse methods to parse in utility class")
```

Comment:
Can probably narrow this to only use the annotations on the methods that do the parsing rather than the whole class.

Reply:
There are many methods around 10 so ig it's better to skip the whole class, this way we can prevent duplication as well and since it is parse util class new methods will likely use parse APIs as well

Reply:
Fair 'nuff.